### ISSUE TITLE
Skip offline cameras

### ISSUE THREAD

**Issue by niemyjski** [blakeblackshear/frigate#182]
I have multiple cameras configured and one of them occassionl reboots for reasons unknown. I noticed when this happens the camera processes will not be all setup and basically hangs and stops all processing. This should be a graceful handle and just not process the feed and retry in a little bit.

Steps to reproduce:
1. Set up 2 or more cameras.
2. Turn off one camera or make sure the ip is invalid in the configuration
3. Startup and see that nothing is processed.

You'll get an error like this:

```
frigate    | Creating ffmpeg process...
frigate    | ffmpeg -hide_banner -loglevel panic -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental -fflags +genpts+discardcorrupt -vsync drop -rtsp_transport http -stimeout 5000000 -use_wallclock_as_timestamps 1 -i rtsp://10.0.1.100:7447/token -f rawvideo -pix_fmt rgb24 pipe:
frigate    | ffprobe -v panic -show_error -show_streams -of json "rtsp://10.0.1.100:7447/token"
frigate    | {'error': {'code': -1094995529, 'string': 'Invalid data found when processing input'}}
frigate    | Traceback (most recent call last):
frigate    |   File "detect_objects.py", line 361, in <module>
frigate    |     main()
frigate    |   File "detect_objects.py", line 204, in main
frigate    |     frame_shape = get_frame_shape(ffmpeg_input)
frigate    |   File "/opt/frigate/frigate/video.py", line 39, in get_frame_shape
frigate    |     video_info = [s for s in info['streams'] if s['codec_type'] == 'video'][0]
frigate    | KeyError: 'streams'
```

Even if this is the second camera and the first cameras process is started, it seems to hang as well and nothing is ever updated or triggered. It would be nice if this was gracefully handled and maybe even the error and status was put into an mqtt topic.

**Comment by jameswarner62** [comment]
This would be a great feature if possible! Under UPS battery conditions only essential cameras are powered up, so if frigate needs to be restarted - it can't pick back up

**Comment by blakeblackshear** [comment]
Is this recovering now?

**Comment by niemyjski** [comment]
I'll start paying a closer attention but I'm using your add-on which hasn't been upgraded to 0.8. I've been resetting it periodically because there didn't see much in the logs after a failure so I assumed it didn't recover.

**Comment by blakeblackshear** [comment]
Reopen if this isn't resolved after the release of 0.8.0.
